### ISSUE TITLE
Why mIoU is 0 in some scenarios

### ISSUE THREAD

**Issue by chenhaomingbob** [unique1i/SceneSplat#4]
Hello, thank you for your excellent work! 

I encountered some confusion when testing on the ScanNet dataset. Could you please take a look?
Specifically, when I was testing the performance of your language pre-trained model, the mIoU is 0 in some scenarios. 

Below is my log:
In scenes `scene0025_01`, `scene0046_00`, `scene0025_00`  and `scene0030_00`, miou is 0 in all cases. Only a few scenes have correct results, such as `scene046_02`, where miou is 0.37.
<img width="1616" height="988" alt="Image" src="https://github.com/user-attachments/assets/17c77790-09f6-4d18-834b-c75774581ebb" />

This is my run script:
`python tools/train.py --config-file /data1/chm/codehub/2025/SceneSplat/configs/scannet/lang-pretrain-scannet-mcmc-wo-normal-contrastive.py --options weight=./model_best.pth test_only=true gpu_num=1 batch_size_val=1 batch_size_test=1 num_worker=4`

I download the `model_best.pth` from https://huggingface.co/GaussianWorld/SceneSplat_lang-pretrain-concat-scan-ppv2-matt-mcmc-wo-normal-contrastive

**Comment by chenhaomingbob** [comment]
Sorry, I found the reason. It turned out that it was reading the results of my previous incorrect run. 
I cleared the results in the `exp` folder, and when I ran it again, I didn't see the miou=0.

**Comment by unique1i** [comment]
Thanks for pointing this out! Yes, the tester will try to load the existing predictions.

Here I attach the inference log for ScanNet20 and ScanNet200, which has the per-scene IoU and accuracy. I find only for the `scene423_01` and `scene423_02` in ScanNet200 evaluation, the IoU and accuracy are 0. For these two scenes, there are only 3 classes in the GT labels, floor, armchair, and coffee table. The model indeed made mistakes, e.g., armchair as sofa chair, and coffee table as stool. Note, the GT labels also have inconsistency comparing to `scene423_00`.

Apart from these two scenes, the zero-shot segmentation performance is as expected.

[eval.log](https://github.com/user-attachments/files/21792420/eval.log)
